apps/src/sites/studio/pages/teacher_dashboard/index.js

```diff
@@ -27,6 +27,8 @@ const baseUrl = `/teacher_dashboard/sections/${section.id}`;
 $(document).ready(function () {
   registerReducers({teacherSections, sectionData, manageStudents, sectionProgress, scriptSelection, stats, textResponses, sectionAssessments});
   const store = getStore();
+  store.dispatch(asyncSetSections());
```

Comment:
Is there a reason we're getting the section data asynchronously from `/dashboardapi/sections` instead of including it in the initial view as part of `scriptData`, given we will always need it?

Reply:
nope, that's a much better idea!